src/pages/individual/EditIndividualMetadata.jsx

```diff
@@ -179,42 +179,46 @@ export default function EditIndividualMetadata({
             setFormErrors(requiredFieldErrors);
             if (requiredFieldErrors.length > 0) return;
 
-            // Always include the defaultName. If it does not already exist, add it.
+            // Always include the firstName. If it does not already exist, add it.
             // Otherwise, replace it.
-            const defaultNameGuid = deriveNameGuid(
+            const firstNameGuid = deriveNameGuid(
               metadata,
-              'defaultName',
+              'FirstName',
             );
-            const defaultNameProperty = defaultNameGuid
+            const firstNameProperty = firstNameGuid
               ? {
                   op: 'replace',
-                  value: defaultFieldValues.defaultName,
-                  guid: defaultNameGuid,
+                  value: defaultFieldValues.firstName,
+                  guid: firstNameGuid,
                 }
               : {
                   op: 'add',
-                  value: defaultFieldValues.defaultName,
-                  context: 'defaultName',
+                  value: defaultFieldValues.firstName,
+                  context: 'FirstName',
                 };
 
-            const names = [defaultNameProperty];
+            const names = [firstNameProperty];
 
-            // If there was a nickname, update it with whatever the value is now.
-            // If there was not a nickname, but there is a value now, add it.
-            // If a nickname did not already exist, don't add it.
-            const nicknameGuid = deriveNameGuid(metadata, 'nickname');
-            const nicknameFieldValue = defaultFieldValues.nickname;
-            if (nicknameGuid) {
+            // If there was an adoptionName, update it with whatever the value is now.
+            // If there was not an adoptionName, but there is a value now, add it.
+            // If an adoptionName did not already exist, don't add it.
+            const adoptionNameGuid = deriveNameGuid(
+              metadata,
+              'AdoptionName',
+            );
+            const adoptionNameFieldValue =
+              defaultFieldValues.adoptionName;
```

Comment:
I didn't quite follow how this is a guaranteed non-null?

Reply:
The way I see it, defaultFieldValues is initially set on declaration to the return value from getInitialFormValues, which will be an object. defaultFieldValues is only ever reassigned in the onChange for the edit component. The value that it is set to is at least an empty object. I don't see a way that defaultFieldValues could ever be null, but I may have missed something.